### PR TITLE
Update faq.md

### PR DESCRIPTION
**Pull request by slhenry**

### CODE REVIEW

File: content/faq.md [hunk 1]
```diff
@@ -89,12 +87,17 @@ To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can u
 
 WCAG 2.0 and WCAG 2.1 are both existing standards. WCAG 2.1 does not deprecate or supersede WCAG 2.0. W3C encourages you to use the most recent version of WCAG when developing or updating content or accessibility policies.
 
-## What's next after WCAG 2.1? {#next}
+## What's next for WCAG? What about “Silver”? {#next}
+
+The Accessibility Guidelines Working Group (AG WG) is developing WCAG 2.2 in 2020. The structure and content of WCAG 2.2 is the same as 2.1 and 2.0. Version 2.2 will include new accessibility requirements, called “success criteria”. You can review proposed additional success criteria in the [in-progress, unpublished “Editors’ Draft” of WCAG 2.3](https://w3c.github.io/wcag/guidelines/22/).
+
+AG WG is currently not planning to publish another version of WCAG 2, that is, not do a WCAG 2.3 — although that might change.
```
Comment (alastc) [inline content/faq.md:94]: Could this be softened a bit, e.g. "AG WG has not decided whether to publish another version of the WCAG 2, that is, whether to do a WCAG 2.3."

File: content/faq.md [hunk 2]
```diff
@@ -89,12 +87,17 @@ To put it another way: If you want to meet both WCAG 2.0 and WCAG 2.1, you can u
...
+AG WG is working on a different accessibility standard that is broader than web content. It was previously temporarily referred to as “Silver”. In-progress, un-approved [requirements for the new standard](https://w3c.github.io/silver/requirements/) are available. The new standard is expected after 2022.
 
-In the interim, the Working Group might decide to develop WCAG 2.2 to provide additional updates to address current accessibility requirements.
+Naming: The new standard is currently referred to as “W3C Accessibility Guidelines (WCAG) 3.0”. After considerable consideration, this name was chosen because of wide-spread familiarity with the “WCAG” acronym, and to encompass the broader scope beyond “content”.
 
+We will announce when more information is available on WCAG 2.3 and WCAG 3.0. To get announcements of updated drafts for review in e-mail, tweets, and RSS, see [Get WAI News](https://www.w3.org/WAI/news/subscribe/).
```
Comment (alastc) [inline content/faq.md:100]: Should that be 2.2 and WCAG 3.0?

File: content/faq.md (hunk 1 above)
Comment (slhenry) [reply content/faq.md]: Thank your for the more accurate info! I simplified it even more to:
> AG WG has not decided whether or not to do another version of WCAG 2 (WCAG 2.3).

File: content/faq.md (hunk 2 above)
Comment (slhenry) [reply content/faq.md]: yup. fixed. thanks for the catch!

**Pull request merged**